src/etc/inc/openvpn.inc

```diff
@@ -447,6 +447,16 @@ function openvpn_get_digestlist() {
 	return $digests;
 }
 
+function openvpn_get_keydirlist() {
+	$keydirs = array(
+		'-1' => gettext('Use default keydir direction'),
```

Comment:
I'd rather have an option to preserve the default behavior rather than assuming it in the GUI. It is safer than forcing everything to have an explicit option. We can't risk breaking any existing configurations.